packages/ui-kit/src/components/Select/Select.tsx

```diff
@@ -0,0 +1,102 @@
+import { ClickAwayListener } from '@mui/base/ClickAwayListener'
+import { Select as MUISelect, SelectProps as MUISelectProps } from '@mui/base/Select'
+import { SelectOptionDefinition, useSelect } from '@mui/base/useSelect'
+import { prepareForSlot } from '@mui/base/utils'
+import classNames from 'classnames'
+import * as React from 'react'
+import { Button, ButtonProps } from '../Button'
+import { ChevronDownIcon } from '../Icons/ChevronDown'
+import { Option } from './Option'
+import componentStyles from './Select.module.scss'
+
+export interface SelectProps
+  extends Pick<ButtonProps, 'startAdornment' | 'endAdornment' | 'size'>,
+    MUISelectProps<string, false> {
+  options?: SelectOptionDefinition<string>[]
+}
+
+const ButtonSlot = prepareForSlot(Button)
+
+export const Select = React.forwardRef<HTMLButtonElement, SelectProps>(
```

Comment:
Is this supporting keyboard navigation? Do we get that from MUI, or do we need to implement & test ourselves?

Reply:
That's a good catch, thanks! It worked somehow but with a bug. I've refactored the component.